packages/@lwc/template-compiler/src/__tests__/fixtures/parsing-errors/forbidden-embed-tag/metadata.json

```diff
@@ -0,0 +1,26 @@
+{
+    "warnings": [
+        {
+            "code": 1051,
+            "message": "LWC1051: Forbidden tag found in template: '<embed>' tag is not allowed.",
+            "level": 1,
+            "location": {
+                "line": 2,
+                "column": 5,
+                "start": 15,
+                "length": 63
+            }
+        },
+        {
+            "code": 1050,
+            "message": "LWC1050: Forbidden MathML namespace tag found in template: '<embed>' tag is not allowed within <math>",
```

Comment:
This seems redundant since the other restriction is more strict.

Reply:
1. The other restriction is only applied when the <embed> tag is under a xhtml namespaced tag. https://github.com/salesforce/lwc/blob/master/packages/@lwc/template-compiler/src/parser/index.ts#L867
2. I have to rollback the restriction added in this PR in HTML_TAG_BLACKLIST. There are folks still using `<object>` in components.